java/api/src/main/java/io/ray/api/runtimecontext/Address.java

```diff
@@ -0,0 +1,35 @@
+package io.ray.api.runtimecontext;
+
+import io.ray.api.id.UniqueId;
+import java.util.Objects;
+
+public class Address {
+  public final UniqueId nodeId;
+  public final String ip;
+  public final int port;
+
+  public Address(UniqueId nodeId, String ip, int port) {
+    this.nodeId = nodeId;
+    this.ip = ip;
+    this.port = port;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
Why we need a API in which the input type is Object instead of Address?

Reply:
This is an overloaded equal method of the object class used to compare two objects for equality.